.travis.yml

```diff
@@ -22,6 +22,7 @@ env:
     - ROS_REPO=ros
     - BUILDER=colcon
     - CATKIN_LINT=pedantic
+    - AFTER_INSTALL_TARGET_DEPENDENCIES='pip install numpy_ringbuffer'
```

Comment:
That appears to not be possible since this is not a standard package that is contained in apt (or something like that @Olavhaasie may be able to explain better)

Reply:
All packages in the `package.xml` are installed by the `rosdep` tool. `rosdep` has [a list](https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml) of packages that it knows how to install. Sadly `numpy-ringbuffer` is not one of them. You could create a PR there to add it.

Reply:
I have indeed seen the list but u could chose to use the collection library, it has a similar function.